sdk/storage/Azure.Storage.Blobs/tests/BlobClientTests.cs

```diff
@@ -742,6 +743,35 @@ public async Task UploadAsync_OverwritesDeliberately_Path()
                 }
             }
         }
+
+        [LiveOnly]
+        [Test]
+        public async Task UploadAsync_ProgressReporting()
+        {
+            // Arrange
+            await using DisposingContainer test = await GetTestContainerAsync();
+            BlobClient blob = InstrumentClient(test.Container.GetBlobClient(GetNewBlobName()));
+            Progress progress = new Progress();
+            using var stream = new MemoryStream(GetRandomBuffer(Constants.GB));
+
+            // Act
+            await blob.UploadAsync(content: stream, progressHandler: progress);
+
+            // Assert
```

Comment:
I think we should have an Assertion that the final progress number is Constants.GB.

Reply:
fixed